### DISCUSSION TITLE
I'd like an `.RPM` installation file.

### DISCUSSION THREAD

**Discussion by roke-julian-lockhart** [PrismLauncher/PrismLauncher#2846] [Ideas]
**Rationale**

Per https://discussion.fedoraproject.org/t/gnome-abrt-cant-upload-crash-reports-from-flatpak-apps/66462/6, GNOME-ABRT can't report crashes of Flatpak applications, so it would make reporting the occasional crash significantly easier.

**Alternatives**

1. Although https://github.com/PrismLauncher/prismlauncher.org/blob/483fc4b63088c8c5c14c97a52df73b0c13119b84/src/wiki/development/build-instructions/linux.md#building-an-rpm--for-fedora exists, it's not trivial.
1. The Flatpak is as aforementioned.

**Solutions**

1. I'd consider a [CPOR repository](https://docs.pagure.org/copr.copr/#copr-buildsystem:~:text=Copr%20is%20designed%20to%20be%20a%20lightweight%20buildsystem%20that%20allows%20contributors%20to%20create%20packages%2C%20put%20them%20in%20repositories%2C%20and%20make%20it%20easy%20for%20users%20to%20install%20the%20packages%20onto%20their%20system.%20Within%20the%20Fedora%20Project%20it%20is%20used%20to%20allow%20packagers%20to%20create%20third%20party%20repositories.) to be sufficient.
1. However, it'd be brilliant if installable by default in the official repositories.

**Comment by Ari-43** [comment]
Would running it inside a distrobox, toolbox, or other container work for this use case? 

An RPM in either the official repository or COPR would be nice, but running it in a container may be a decent temporary solution.

**Reply by roke-julian-lockhart** [reply]
> https://github.com/orgs/PrismLauncher/discussions/2846#discussioncomment-10707328

@Ari-RERA-43, I wouldn't mind using `distrobox` (if `distrobox` does work for my purposes - I presume so, should I install the relevant symbol packages in the container) but I don't think that Prism provides *any* native packages for any Linux-based OS. macOS is the sole exception, by necessity.

**Reply by Ari-43** [reply]
> > https://github.com/orgs/PrismLauncher/discussions/2846#discussioncomment-10707328
> 
> 
> 
> @Ari-RERA-43, I wouldn't mind using `distrobox` (if `distrobox` does work for my purposes - I presume so, should I install the relevant symbol packages in the container) but I don't think that Prism provides *any* native packages for any Linux-based OS. macOS is the sole exception, by necessity.

Unfortunately I don't know enough about ABRT to know what would be required to get it to work with a container. However, after looking back at the website, it appears that PrismLauncher does actually have [an RPM available through Copr](https://prismlauncher.org/download/?from=button#centos-stream-fedora-red-hat-enterprise-linux).

**Reply by roke-julian-lockhart** [reply]
> https://github.com/orgs/PrismLauncher/discussions/2846#discussioncomment-10707416

@Ari-RERA-43, you genius! Thank you. That was unexpectedly hidden.

**Comment by JulianVennen** [comment]
There already is a COPR repo: https://copr.fedorainfracloud.org/coprs/g3tchoo/prismlauncher/
This is also referenced on https://prismlauncher.org/download/

**Reply by roke-julian-lockhart** [reply]
> https://github.com/orgs/PrismLauncher/discussions/2846#discussioncomment-10707567

@JulianVennen, thanks. Switched to https://github.com/orgs/PrismLauncher/discussions/categories/q-a and accepted this comment as the answer.
